Templates/Multiplayer/Template/Gem/Code/Source/Spawners/RoundRobinSpawner.cpp

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
- * 
+ *
  * SPDX-License-Identifier: Apache-2.0 OR MIT
  *
```

Comment:
Might be worth taking a look at this on github here and seeing all the whitespace-only changes... its not a big deal but it makes it cleaner if whitespace only changes are reverted.

Reply:
I waffled a bit about that a bit, myself, but ended up erring on the side of leaving them, because it complies with the style guide (as enforced by the `.editorconfig` in the main repo). I can pull them out, though.